components/docs-chef-io/content/automate/ha_troubleshooting.md

```diff
@@ -180,4 +180,23 @@ terraform taint module.bootstrap_automate.null_resource.automate_post[1]
 terraform taint module.bootstrap_automate.null_resource.automate_post[2]
 ```
 
+### Issue: Deployment or Upgrade Fails as UnknownError
 
+While trying to deploy or upgrade Chef Automate HA, if the command did not exit gracefully
+
+The possible error looks like as shown below:
+
+```bash
+      ┬ Exiting as execution process completed
+UnknownError: Unable to determine error cause: Command did not exit gracefully
+```
+
+- Examine the logs from bastion (/hab/a2_deploy_workspace/logs/a2ha-run.log) or from frontend/backend nodes in svc-load.log (or) automate-ctl.log in /hab/var/automate-ha/
+- If you found that `sysctl: command not found`
+- Follow the below steps
```

Comment:
If you find `sysctl: command not found`, follow the step below:

(merge these two pointers)

Reply:
Done